index.html

```diff
@@ -283,9 +283,9 @@ <h2 class="lang-h2">Skills</h2>
         <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
             please get in touch.</h2>
         <form class="form">
-            <input class="fullname" type="text" name="fullname" placeholder="Full name">
-            <input class="fullname" type="text" name="fullname" placeholder="Email address">
-            <textarea class="text-area" placeholder="Enter text here"></textarea>
+            <input class="fullname" type="text" name="fullname" placeholder="Full name" maxlength="30" required >
+            <input class="fullname" type="email" name="fullname" placeholder="Email address" required>
+            <textarea class="text-area" placeholder="Enter text here" maxlength="500"></textarea>
```

Comment:
- [ ] Kindly add the required attribute to the `textarea` so validation is set on it and users must be able to write you a message before the form can be set.